force-app/main/default/lwc/rd2StatusAutomationSettings/rd2StatusAutomationSettings.js

```diff
@@ -0,0 +1,205 @@
+import daysForClosedTooSmall from '@salesforce/label/c.RD2_StatusAutomationInvalidNumberOfDaysForClosed';
+import daysForLapsedTooLarge from '@salesforce/label/c.RD2_StatusAutomationInvalidNumberOfDaysForLapsed';
+
+const STATES = {
+    LOADING: 'LOADING',
+    READY: 'READY',
+    ERROR: 'ERROR',
+    RD2_DISABLED: 'RD2_DISABLED'
+};
+
+export default class Rd2StatusAutomationSettings extends LightningElement {
+    labels = {
+        statusAutomationIntro,
+        configurationSectionHeader,
+        closedDefinition,
+        lapsedDefinition,
+        daysForClosed,
+        daysForLapsed,
+        setClosedStatus,
+        setLapsedStatus,
+        editButtonLabel,
+        cancelButtonLabel,
+        saveButtonLabel,
+        rd2DisabledError,
+        commonError,
+        commonNoneSpecific,
+        daysForClosedTooSmall,
+        daysForLapsedTooLarge
+    };
+
+    displayState = STATES.LOADING;
+    errorMessage;
+
+    isEdit = false;
+
+    automationSettings;
+    numberOfDaysForLapsed;
+    numberOfDaysForClosed;
+    closedStatus;
+    lapsedStatus;
+    lapsedStatusOption;
+    closedStatusOption;
+
+    inputDaysForLapsed;
+    inputDaysForClosed;
+
+    maximumDaysForLapsed;
+    minimunDaysForClosed;
+
+    inputClosedStatus;
+    inputLapsedStatus;
+
+
+    async connectedCallback() {
+        try {
+            this.automationSettings = await getAutomationSettings();
+
+            if (this.automationSettings.rd2Enabled) {
+                this.assignSettings(this.automationSettings);
+            } else {
+                this.displayState = STATES.RD2_DISABLED;
+            }
+
+        } catch (ex) {
+            this.displayState = STATES.ERROR;
+            this.errorMessage = ex.body.message;
+        }
+    }
+
+    assignSettings(automationSettings) {
+        this.numberOfDaysForClosed = (automationSettings.numberOfDaysForClosed) 
+            ? automationSettings.numberOfDaysForClosed
+            : this.labels.commonNoneSpecific;
+        this.inputDaysForClosed = automationSettings.numberOfDaysForClosed;
+        this.maximumDaysForLapsed = (this.numberOfDaysForClosed == null)
+            ? null
+            : parseInt(this.numberOfDaysForClosed) - 1;
+    
+        this.numberOfDaysForLapsed = (automationSettings.numberOfDaysForLapsed)
+            ? automationSettings.numberOfDaysForLapsed
+            : this.labels.commonNoneSpecific
+        this.inputDaysForLapsed = automationSettings.numberOfDaysForLapsed;
+        this.minimunDaysForClosed = (this.numberOfDaysForLapsed == null)
+            ? null
+            : parseInt(this.numberOfDaysForLapsed) + 1;
+
+        this.closedStatus = automationSettings.closedStatus;
+        this.inputClosedStatus = automationSettings.closedStatus;
+
+        this.lapsedStatus = automationSettings.lapsedStatus;
+        this.inputLapsedStatus = automationSettings.lapsedStatus;
+
+        this.closedStatusOption = automationSettings.closedStatusOption;
```

Comment:
Edge case: if you setup a new Status to State Mapping, then go to the Status Automation page without refreshing, those states won't show up.

Reply:
This is an interesting one. I will test around to see if I can somehow refresh the Automation component after Status Mapping component is saved.